<launch>
    <node pkg="mouse_teleop" type="mouse_teleop.py" name="mouse_teleop" output="screen"/>

    <node pkg="rc110_twist_to_ackermann" type="rc110_twist_to_ackermann" name="twist_to_ackermann_mouse">
        <remap from="cmd_vel" to="mouse_vel" />
        <remap from="drive_ad" to="drive_manual" />
    </node>
</launch>
